UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs

```diff
@@ -162,7 +162,7 @@ public IActionResult CommitBatch([FromRoute] string batchId, [FromBody] List<Bat
         {
             if (!string.IsNullOrEmpty(batchId) && !string.IsNullOrEmpty(fileName))
```

Comment:
`!string.IsNullOrEmpty(fileName)` no longer needed

Reply:
`if (!string.IsNullOrEmpty(batchId))` is still needed

Reply:
Yes. Right did changes accordingly. Thanks!